R/geom-boxplot.r

```diff
@@ -49,7 +49,7 @@
 #'   if the notches of two boxes do not overlap, this suggests that the medians
 #'   are significantly different.
 #' @param notchwidth For a notched box plot, width of the notch relative to
-#'   the body (default 0.5)
+#'   the body (default = `0.5`)
```

Comment:
~~Are we surrounding just the value of an argument with backticks? (I'm genuinely asking—if so, I think we're inconsistent)~~
Move backticks to surround argument and value.

Reply:
I like defaults to `notchwidth = 0.5` — though, honestly, both make sense. Go with where the spirit moves you!

Reply:
Done!